ws.py

```diff
@@ -91,6 +92,13 @@
 if not GlobalVars.metasmoke_ws_host:
     log('info', "No metasmoke websocket host found, which is okay if you're anti-websocket")
 
+# Initiate DNS
```

Comment:
Having this at this location in the file will probably be fine. However, we should make sure that we're not relying on the configuration choices having been applied at the time the other packages which use the `dns,resolver` are imported (i.e. it won't have the configuration applied at the time those packages initialize).

That's not a problem. It's just something which we should be aware of to make sure it really isn't a problem.

Reply:
Currently only chatcommands and findspam rely on dns.resolver to function.  And chatcommands' `dig` actually creates a brand new Resolver and not default resolver to return IP data.  Done this way to make sure that we get 'good' DNS and not crap DNS in the event we are digging to debug odd IP values.